scarb/src/compiler/plugin/proc_macro/host.rs

```diff
@@ -133,7 +133,7 @@ impl ProcMacroHostPlugin {
                     .collect_vec()
             })
             .collect::<Vec<_>>();
-        expansions.sort_unstable_by_key(|e| e.expansion.name.clone());
+        expansions.sort_unstable_by_key(|e| (e.package_id, e.expansion.name.clone()));
```

Comment:
Shouldn't this be the other way round? 🤔 We need identical names to be one after another to catch duplicates

Reply:
bump